latch_cli/nextflow/workflow.py

```diff
@@ -156,11 +160,27 @@ def generate_nextflow_workflow(
         else:
             no_defaults.append(sig)
 
+        if param.samplesheet:
+            construct_samplesheet_import = (
+                "from latch_metadata.parameters import construct_samplesheet"
```

Comment:
might be worth adding a try catch here so that we can print a more descriptive error:

"""
Failed to import `construct_samplesheet`. Please define a `construct_samplesheet` function in your `latch_metadata` which converts the samplesheet input parameters to a csv. 
"""